spring-integration-core/src/main/java/org/springframework/integration/codec/CompositeCodec.java

```diff
@@ -22,16 +22,28 @@
 import java.io.OutputStream;
 import java.util.HashMap;
 import java.util.Map;
-
-import org.jspecify.annotations.Nullable;
+import java.util.Set;
```

Comment:
I’m not sure how it works right now, but in the past it failed for us when is used only in JavaDocs. Or use FQCN in the link for the method, or mention that method without argument types.